dhis-2/dhis-api/src/test/java/org/hisp/dhis/category/CategoryOptionTest.java

```diff
@@ -152,20 +150,12 @@ private static Stream<Arguments> categoryOptionEqualsParams() {
     boolean isNotEqual = false;
 
     return Stream.of(
-        Arguments.of("name", "uid", "code", "shortName", "description", null, isEqual),
-        Arguments.of("name", "uid", "code", "shortName", "description diff", null, isNotEqual),
-        Arguments.of("name", "uid", "code", "shortName diff", "description", null, isNotEqual),
-        Arguments.of("name", "uid", "code diff", "shortName", "description", null, isNotEqual),
-        Arguments.of("name", "uid diff", "code", "shortName", "description", null, isNotEqual),
-        Arguments.of("name diff", "uid", "code", "shortName", "description", null, isNotEqual),
-        Arguments.of(
-            "name",
-            "uid",
-            "code",
-            "shortName",
-            "description",
-            QueryModifiers.builder().build(),
-            false));
+        Arguments.of("name", "uid", "code", "shortName", null, isEqual),
+        Arguments.of("name", "uid", "code", "shortName diff", null, isNotEqual),
+        Arguments.of("name", "uid", "code diff", "shortName", null, isNotEqual),
+        Arguments.of("name", "uid diff", "code", "shortName", null, isNotEqual),
+        Arguments.of("name diff", "uid", "code", "shortName", null, isNotEqual),
+        Arguments.of("name", "uid", "code", "shortName", QueryModifiers.builder().build(), false));
```

Comment:
I removed the `description` from the `equals` and `hashcode` method in `CategoryOption` class, therefore this test needs to be updated.